components/arm/universalrobots/ur.go

```diff
@@ -383,6 +384,25 @@ func (ua *urArm) MoveToJointPositions(ctx context.Context, joints []referencefra
 	return ua.moveToJointPositionRadians(ctx, referenceframe.InputsToFloats(joints))
 }
 
+func (ua *urArm) MoveThroughJointPositions(
+	ctx context.Context,
+	positions [][]referenceframe.Input,
+	_ *arm.MoveOptions,
```

Comment:
I am assuming this is purposefully left as `_` since we are moving the driver into a module?